nukebuild/numerge.config

```diff
@@ -23,6 +23,19 @@
           "DoNotMergeDependencies": true
         }
       ]
+    },
+    {
+      "Id": "Avalonia.Win32",
+      "MergeAll": true,
```

Comment:
IIRC MergeAll means "merge all dependencies", which made sense for the Avalonia package since there were quite a lot of them.

For Avalonia.Win32 it can be false since Avalonia.Win32.Automation is specified explicitly anyway.

Otherwise we might have problems with Avalonia.OpenGl/Avalonia.Vulkan

Reply:
Avalonia.OpenGl/Avalonia.Vulkan were not included, as I double checked. But still makes sense to set it to false.